packages/core/src/messages/packet/index.spec.ts

```diff
@@ -90,7 +84,7 @@ describe('test packet composition and decomposition', function () {
 
         await nodes[a].paymentChannels.channel.create(
           nodes[b].getId().pubKey.marshal(),
-          async () => nodes[b].getId().pubKey.marshal(),
+          undefined, //async () => nodes[b].getId().pubKey.marshal(),
```

Comment:
@nionis Can you take a look at this change?

Reply:
this api should probably change.

Reply:
yeah it seems redundant, I think this was added for Polkadot, cc @robertkiel 
we don't use `async () => nodes[b].getId().pubKey.marshal()` in `core-ethereum`